src/views/tables/Orderbook.tsx

```diff
@@ -194,7 +231,7 @@ const OrderbookTable = ({
       label="Orderbook"
       data={data}
       columns={columns}
-      getRowKey={(row: RowData) => `${row.side}-${row.price}`}
+      getRowKey={(row: RowData) => row.key}
       getRowAttributes={(row: RowData) => ({
```

Comment:
nit: I wonder if we can remove the row hover/cursor styling for the dummy rows here.

Reply:
Yeah good call out! I added an additional css var to determine pointer-events on the `tr`